ash/src/extensions/khr/pipeline_executable_properties.rs

```diff
@@ -7,23 +7,16 @@ use std::mem;
 #[derive(Clone)]
 pub struct PipelineExecutableProperties {
     handle: vk::Instance,
-    pipeline_executable_properties_fn: vk::KhrPipelineExecutablePropertiesFn,
+    fns: vk::KhrPipelineExecutablePropertiesFn,
 }
 
 impl PipelineExecutableProperties {
     pub fn new(entry: &Entry, instance: &Instance) -> Self {
-        let pipeline_executable_properties_fn =
-            vk::KhrPipelineExecutablePropertiesFn::load(|name| unsafe {
-                mem::transmute(entry.get_instance_proc_addr(instance.handle(), name.as_ptr()))
-            });
-        Self {
-            handle: instance.handle(),
-            pipeline_executable_properties_fn,
-        }
-    }
-
-    pub fn name() -> &'static CStr {
-        vk::KhrPipelineExecutablePropertiesFn::name()
+        let handle = instance.handle();
+        let fns = vk::KhrPipelineExecutablePropertiesFn::load(|name| unsafe {
+            mem::transmute(entry.get_instance_proc_addr(handle, name.as_ptr()))
```

Comment:
It looks like `PipelineExecutableProperties` uses the `device` in every call so feel free to switch this all to use the device. Fine to do it in a separate PR in a way that gives the least amount of work/conflicts for you with this PR :)

Reply:
I decided to document this and take care of it (also `TimelineSemaphore`) in #499 :wink: